### ISSUE TITLE
using the lib with braintree's 3DSecure

### ISSUE THREAD

**Issue by darkbercislaw** [nathanstitt/react-braintree-fields#13]
Hi,

I'd like to use braintree's 3DSecure with `react-braintree-fields` lib. The library itself does not support it but I can use the `braintree-web/three-d-secure` package directly. To create an instance of the threeDSecure the braintree client instance is required. Is there a possibility to obtain it? I don't see it being available on the Braintree or HostedField components.

Regards,
Robert

**Comment by nathanstitt** [comment]
Hi @darkbercislaw apologies for the extremely late reply, this kind of fell of my radar and I wasn't getting emails.

I've just released version 1.2 that exposes the Braintree api instance.

You can obtain it by adding getting the reference to the Braintree component and then referencing `api.client` on it.

For instance if the ref is set something like in the demo: https://github.com/nathanstitt/react-braintree-fields/blob/master/demo.jsx#L83

You could add a:

```jsx
    componentDidMount() {
       const { client } = this.braintree.api;
       client.verify3DS(options);
    }

```

Hope that helps, please let me know if it doesn't

**Comment by NguyenTheSon** [comment]
Hi @nathanstitt I don't see verify3DS method in client. Can you support me?
![image](https://user-images.githubusercontent.com/12813174/127288349-81e42e7c-eafe-425b-ba12-5efd83d93e1a.png)
